sidecar/config/config.go

```diff
@@ -146,6 +159,9 @@ type Config struct {
 	Supervise bool `yaml:"supervise"`
 
 	HealthChecks []HealthCheck `yaml:"healthchecks"`
+	HealthChecksCACertPath string `yaml:"healthchecks_ca_cert_path"`
```

Comment:
I'm alright either way. Removing `A8_HEALTHCHECKS_CA_CERT_PATH` does make things simpler, but I can see why @paulben added it for convenience.